### DISCUSSION TITLE
Curious to learn how to run a local copy of Symbiota Docs? Check out this quick guide!

### DISCUSSION THREAD

**Discussion by arbolitoloco** [Symbiota/community-board#32] [Show and tell]
You can run the Symbiota Docs website in your own computer if you’d like to have it offline, if you'd like to contribute with content and are interested in previewing changes before pushing them to us, or for development purposes as well, in case you’d like to suggest theme modifications.

This quick guide explains how to do it: 🔗 https://biokic.github.io/symbiota-docs/contribute/local_version/

Feel free to post questions or feedback in this thread.

**Comment by lalrochhara** [comment]
> 

Is symbiota possible to host using github.io page? I mean, trying to run here 
:  [Famhawite Infosys](https://famhawiteinfosys.github.io) it's look like this, it's not same.

**Comment by themerekat** [comment]
Not Symbiota by itself, but you can create a local version of Symbiota docs, our documentation site, for editing and contributing.
